### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by lalithsuresh**
Continuous integration for Hiero. @mbudiu-vmw will need to create an account on travis-ci.org and enable CI for github.com/mbuidiu-vmw/hiero. Here's a sample of what the result looks like: https://travis-ci.org/lalithsuresh/hiero/builds/227475607

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/lalithsuresh/hiero.svg?branch=master)](https://travis-ci.org/lalithsuresh/hiero)
```
Comment (mbudiu-vmw) [inline README.md]: Seems like you already have an account. Can't we just keep using yours?

File: hieroplatform/hieroplatform.iml [hunk 2]
```diff
@@ -1,16 +1,46 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_1_8" inherit-compiler-output="false">
+  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_1_8">
```
Comment (mbudiu-vmw) [inline hieroplatform/hieroplatform.iml]: Looks to me like this belongs to another PR.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+language: java
```
Comment (mbudiu-vmw) [inline .travis.yml]: I don't get it, why is it so easy to do this on Travis and so hard for our users to install of the software?

File: README.md (hunk 1 above)
Comment (lalithsuresh) [inline README.md]: No, that URL shows the build status of my fork of the hiero repo. We need to show the status for the main development repo, which is yours.

File: hieroplatform/hieroplatform.iml (hunk 2 above)
Comment (lalithsuresh) [inline hieroplatform/hieroplatform.iml]: Yeah, I'll clean that up.

File: .travis.yml (hunk 3 above)
Comment (lalithsuresh) [inline .travis.yml]: I'm only building hieroplatform here and running its tests. I think users are having trouble setting up tomcat.

File: README.md (hunk 1 above)
Comment (mbudiu-vmw) [inline README.md]: You are a commiter, so why is that necessary?
Should we wait until we get a repo in the vmware org?
Also, the UI part has no unit tests :( But we can try to at least build it.

Comment (lalithsuresh) [inline README.md]: We could wait until we get a vmware org repo. That seems appropriate. I'll see if my login works for this.

**Comment by mbudiu-vmw** [comment]
@lalithsuresh : let's do this.
Can you bring this up-to-date and also add the web project to Travis?
